src/components/navBar/translationDropdown.tsx

```diff
@@ -0,0 +1,37 @@
+import React from 'react';
+import { Dropdown, MenuProps } from 'antd';
+import i18n from 'i18next';
+import { GlobalOutlined } from '@ant-design/icons';
+import styled from 'styled-components';
+
+const items: MenuProps['items'] = [
+  {
+    key: 'en',
+    label: 'English',
+  },
+  {
+    key: 'es',
+    label: 'Español',
+  },
+];
```

Comment:
Will we be easily able to add other languages to this array? Does this just use the i18 language library/autotranslate?

Reply:
Yep! As long we have the translations for other languages, we can just add their key and appropriate label for languages here. We'll just need to make sure that the key we specify here matches what we use in `src/i18n/i18n.ts` where we map languages to translation files.